qt/aqt/utils.py

```diff
@@ -638,6 +638,23 @@ def accept() -> None:
     return ret[0] if ret else None
 
 
+def checkNeedPortalSupport():
```

Comment:
Minor nit: we try to use PEP8 snake_case in new code. It's an easy mistake to make when the surrounding functions are still camelCase; I've fallen into it a number of times myself.

Suggestion: how about running_in_sandbox() instead, or something similar?

Reply:
Yep, I just copied the format from around where I was working 😅

Yeah, that rename seem great.